src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -9,47 +9,71 @@
  * Feel free to remove this class and create your own.
  */
 public class CalculatorTest {
-    private static Random rn;
     private static Calculator calculator;
 
     @BeforeClass
     public static void before() {
         calculator = new Calculator();
-        rn = new Random();
     }
 
     @Test
     public void addingTest() {
-        int a = rn.nextInt(), b = rn.nextInt();
+        int a = 5, b = 3;
```

Comment:
```suggestion
        int a = 5; 
        int b = 3;
```
https://mate-academy.github.io/style-guides/java/java.html#s4.8.2-variable-declarations